### ISSUE TITLE
Tracking Issue: Accessibility Checks 

### ISSUE THREAD

**Issue by bdougie** [open-sauced/app#358]
We can start with a lighthouse score, but open to suggestions to other tools. 

> @0-vortex @bdougie @pixelsbyeryc @chadstewart Let's drop our view on the current Select component I created before replacing the `Supabase` select with it. I'm still not convinced my current implementation is right praypray

I think this component will do for now. I don't see any real glaring issues with it outside of potential accessibility issues. I'd say we can move forward with this once all the necessary changes are completed and we can take a deeper look at all the components after Hacktoberfest.

_Originally posted by @chadstewart in https://github.com/open-sauced/insights/issues/346#issuecomment-1245297348_

**Comment by 0-vortex** [comment]
Storybook and lighthouse should be doable now. Would argue both on CI like https://github.com/marketplace/actions/lighthouse-ci-action

**Comment by 0-vortex** [comment]
One quick mention about radix accessibility, most components will have a section like this in the docs:
<img width="232" alt="Screenshot 2023-01-04 at 00 14 11" src="https://user-images.githubusercontent.com/237133/210450067-cf853604-f653-4107-a093-c30afa4729f4.png">

What that means is that we cannot magically fix tabbing or some aria-level stuff from our ds/wrapper components and have to add additional props for what's missing, while also having to test each page individually - reckon that after implementing the CI action we can finally use this tracking issue to coordinate closing component a11y once we have a system in place to alert us of where all the errors are located (avoiding doing this manually right now)
